src/app/app.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component } from '@angular/core';
+import { RouterOutlet } from '@angular/router';
+import Rox from 'rox-browser';
+import { flags } from './flags';
+import { CommonModule } from '@angular/common';
+
+
+@Component({
+  selector: 'app-root',
+  standalone: true,
+  imports: [
+    RouterOutlet,
+    CommonModule
+  ],
+  templateUrl: './app.component.html',
+  styleUrl: './app.component.css'
+})
+export class AppComponent {
+  title = 'Example Angular application for CloudBees Feature Management';
+
+  sdkKey = '<INSERT YOUR SDK KEY HERE>';
+  options = {};
+
+  message = flags.message
+  fontColor = flags.fontColor
+  fontSize = flags.fontSize
+  showMessage = flags.showMessage
+
+  ngOnChanges() {
+    this.applyStyle()
+  }
+
+  ngOnInit() {
+    this.initFeatureFlags()
+  }
+
+  initFeatureFlags = async () => { 
+    Rox.register('demo', flags)
```

Comment:
This creates a different set of flags compared to the [react sample app](https://github.com/cloudbees-io/react-fm-example/blob/c4c3d21942da6efa5c3acd5c755eb98a869ced78/src/feature-management/FeatureFlagsProvider.tsx#L29), e.g. `demo.showMessage` vs `showMessage`.
I don't mind which, but lets standardise the flags so people can use exactly the same ones across all the sample apps.

<img width="549" alt="image" src="https://github.com/cloudbees-io/angular-fm-example/assets/42846990/bd4df8fb-5306-4222-8ed6-614ceebcb0b1">

Reply:
updated